src/services/config/config.service.test.ts

```diff
@@ -0,0 +1,36 @@
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { Config } from './config.service';
+
+const test = suite('Config');
```

Comment:
We could add `Config.flush()` in a before.each hook so that every test starts with the same initial state.